------------------------------------------------------------
revno: 336
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Thu 2011-01-27 14:38:22 -0500
message:
  fix the filename of the processed userdata
  
  Change /var/lib/cloud/instance/
     user-data-raw.txt.i 
  to 
     user-data.txt.i
=== modified file 'cloudinit/__init__.py'
--- a/cloudinit/__init__.py
+++ b/cloudinit/__init__.py
@@ -39,7 +39,7 @@
    "sem" : "/sem",
    "boothooks" : "/boothooks",
    "userdata_raw" : "/user-data.txt",
-   "userdata" : "/user-data-raw.txt.i",
+   "userdata" : "/user-data.txt.i",
    "obj_pkl" : "/obj.pkl",
    "cloud_config" : "/cloud-config.txt",
    "datadir" : "/data",
